src/Chirp.Razor/Pages/UserTimeline.cshtml

```diff
@@ -34,7 +34,7 @@
     <div class="pagination">
         @for (int i = 1; i <= Math.Ceiling((double) Model.NumOfCheeps / Model.CheepsPerPage); i++)
         {
-            <a href="/@routeName/?page=@i">@i</a>
+            <a href="/@routeName/?pageNumber=@i">@i</a>
```

Comment:
This refers to the url route that we use which sets the `pageNumber` variable when a get request is made, therefore `OnGet(int pageNumber)`

Reply:
But if I am understanding it correctly then pageNumber isnt the endpoint but rather just the link. 
When i tried changing the link the endpoint did not automatically find /page and therefore couldnt find the page parameter.
Don't know if that makes sense

Reply:
Hahahah I figured it out, page is just a reserved keyword, so if you change the code in the following way
```cs
<div class="pagination">
        @for (int i = 1; i <= Math.Ceiling((double) Model.NumOfCheeps / Model.CheepsPerPage); i++)
        {
            <a href="/?dick=@i">@i</a>
        }
</div>
public ActionResult OnGet(int dick = 1)
{
    Console.WriteLine(dick);
    Cheeps = _service.GetCheeps(dick);
    CheepsPerPage = _service.CheepsPerPage;
    NumOfCheeps = _service.GetCheepCount();
    return Page();
}
```
it works fine.